### PR TITLE
vpnc: switch to maintained fork

### PR DESCRIPTION
**Pull request by alyssais**
###### Motivation for this change

This is the same fork now used by Alpine, Arch, Debian, Gentoo, and
SliTaz.  vpnc-script is no longer bundled, but we already have it
packaged anyway so that's okay.

I'm not able to test this, unfortuantely, but nobody else is actively maintaining it and despite asking around I haven't found anybody using it either.  I'm working on it because I'm working on getting connman to cross-compile, and this is a dependency.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/networking/vpnc/default.nix [hunk 1]
```diff
@@ -1,29 +1,21 @@
...
 
 stdenv.mkDerivation {
-  pname = "vpnc";
-  version = "0.5.3-post-r550";
-  src = fetchsvn {
-    url = "https://svn.unix-ag.uni-kl.de/vpnc";
-    rev = "550";
-    sha256 = "0x4ckfv9lpykwmh28v1kyzz91y1j2v48fi8q5nsawrba4q0wlrls";
+  pname = "vpnc-unstable";
+  version = "2021-11-04";
```
Comment (Lassulus) [inline pkgs/tools/networking/vpnc/default.nix:9]: afaik unstable is added to the version not the pname?

Comment (alyssais) [reply pkgs/tools/networking/vpnc/default.nix]: Right you are! Fixed. :)

Comment (Lassulus) [reply pkgs/tools/networking/vpnc/default.nix]: I actually realized that the whole version argument is not so clear as I initially thought: https://github.com/NixOS/rfcs/pull/107

**Comment by Lassulus** [comment]
Result of `nixpkgs-review pr 163070` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>CuboCore.coretoppings</li>
    <li>connman</li>
    <li>connman-gtk</li>
    <li>connman-ncurses</li>
    <li>connmanFull</li>
    <li>connman_dmenu</li>
    <li>networkmanager-vpnc (gnome.networkmanager-vpnc)</li>
    <li>vpnc</li>
  </ul>
</details>

**Pull request merged**